doc/configuration.rst

```diff
@@ -464,7 +464,9 @@ spark-submit
   Command to run in order to submit spark jobs. Default: spark-submit
 
 spark-master
-  Master url to use for spark-submit. Example: local[*]. Default: yarn-client
+  Master url to use for spark-submit. Example: local[*]. Default: See your Spark configuration
+
+  The deprecated Spark1xJob and PySparl1xJob use yarn-client by default
```

Comment:
Using a deprecated directive would be good here:

```rst
.. deprecated:: 1.0.25
   ``Spark1xJob`` and ``PySparl1xJob`` classes were deprecated. [...]
```

(not sure about the next version number. @erikbern or @Tarrasch know better than me :))